docs/user-guide/ze-usage.md

```diff
@@ -0,0 +1,287 @@
+
+## Sample Use Cases
+
+Review the following use cases to understand how to use Zowe Explorer.
+
+* [Work with Data Sets](#work-with-data-sets)
+* [Work with USS Files](#work-with-uss-files)
+* [Work with jobs](#work-with-jobs)
+* [MVS/TSO Commands](#mvs/tso-commands)
+
+### Work with Data Sets
+
+You can use the following functionalities when interacting with data sets:
+
+* **View data sets and use multiple filters**: You can view multiple data sets simultaneously and apply filters to show specified data sets.
+* **Rename data sets**: You can rename specified data sets.
+* **Copy data sets**: You can copy a specified data sets and memebers.
+* **Download, edit, and upload existing PDS members**: You can instantly pull data sets and data set members from the mainframe, edit them and upload back.
+* **Create and delete data sets and data set members**: Enables you to easily create and delete both data sets and their members.
+* **View and access multiple profiles simultaneously**: Enables to work with data sets from multiple profiles.
+* **Submit a JCL**: You can submit a jcl from a chosen data set.
+
+#### View data sets and use multiple filters
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Select the profile that you want to filter.
+4. Click the **Search Data Sets by Entering Patterns** magnifying glass.
+5. From the drop-down, enter the patterns that you want to filter.  
+  The data sets that match your pattern(s) display in the explorer tree.
+
+**Tip:** To provide multiple filters, separate entries with a comma. You can append or postpend any filter with an \*, which indicates wildcard searching. You cannot enter an \* as the entire pattern.
+<img src="../images/ze/ZE-multiple-search.gif" alt="View Data Set">
+
+#### Refresh the list of data sets
+
+1. Navigate to the explorer tree.
+2. Click **Refresh All** button on the right of the **DATA SETS** explorer bar.
+
+#### Rename data sets
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Select a data set you want to rename.
+4. Right-click the data set and select the **Rename Data Set** option.
+5. Change the name of the data set.
+
+<img src="../images/ze/ZE-rename.gif" alt="Rename Data Set">
+
+#### Copy data sets
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Select a member you want to copy.
+4. Right-click the member and select the **Copy Data Set** option.
+5. Right-click the data set where the member belongs and select the **Paste Data Set** option.
+6. Enter the name of the copied member.
+   <img src="../images/ze/ZE-copy.gif" alt="Copy Data Set">
+
+#### Download, edit, and upload existing PDS members
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Open a profile.  
+4. Click the PDS member (or PS) that you want to download.
```

Comment:
Probably "select the PDS member" here... I have a comment later about consistency with "select" and "click" (and "press," which is used later)

Reply:
Correct! Yes, usually we click **an option** from a UI element, or we click **on smth**  if that's an object that we'll be interacting with.
In this particular case, I'd go for either "**Select** the PDS..." or "**Click on** the PDS..."